metrics.nim

```diff
@@ -85,7 +96,10 @@ when defined(metrics):
       result.add('{')
       var textLabels: seq[string] = @[]
       for i in 0..metric.labels.high:
-        textLabels.add("$#=\"$#\"" % [metric.labels[i], metric.labelValues[i].processLabelValue()])
+        try:
+          textLabels.add("$#=\"$#\"" % [metric.labels[i], metric.labelValues[i].processLabelValue()])
+        except ValueError:
+          textLabels.add(metric.labels[i] & "=error") # whatever
```

Comment:
Hell no! Label values should match a certain pattern and, if they don't, the execution should stop. This is not "whatever" territory. This is Sparta!

https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels

Reply:
there we go, it should conform now